packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -85,7 +87,9 @@ class AppBar extends StatelessWidget {
   /// last action.
   final List<Widget> actions;
 
-  /// A widget to aid in building scroll-based collapsing app bar effects.
+  /// This widget is stacked behind the toolbar and the tabbar and it is not
```

Comment:
I agonized over this. I'm not saying that I got it right, but I did go back and forth. I tried to use appbar, tabbar, and toolbar consistently in places where it wasn't necessary to refer to the class names.